extension.json

```diff
@@ -215,6 +215,9 @@
 				"rights/rights",
 				"lock"
 			]
+		},
+		"ThanksMaxLoadThankedPeriodDays": {
```

Comment:
Any better naming suggestions? Couldn't come up with anything more relevant that sounded cooler